source/contact/index.haml

```diff
@@ -0,0 +1,26 @@
+---
+title: Contact
+---
+- content_for :header do
+  %header.aptible-header.hero-gradient--angled.aptible-header--bottom-grid
+    = partial 'partials/main-nav'
+
+.content
+  .grid-container
+    .centered-header.page-header-basic
+      %h1 How can we help you?
+      %h2.page-header-basic__sub
+        Choose a category below to email the support team so we can get back to you as quickly as possible.
+
+    .contact-cards
+      .grid-container.grid--2up
+        %a.grid-item.card{:href => "/contact/sales"}
+          .card__title.card__title--centered
```

Comment:
@henryhund I used the example site's content for placement holders. Let me know if you have any existing content you would like me to use.